oci/oci.go

```diff
@@ -31,7 +31,7 @@ const (
 	// ContainerStateStopped represents the stopped state of a container
 	ContainerStateStopped = "stopped"
 	// ContainerCreateTimeout represents the value of container creating timeout
-	ContainerCreateTimeout = 10 * time.Second
+	ContainerCreateTimeout = 240 * time.Second
```

Comment:
Of course doesn't kubernetes have a timeout for this also?

Reply:
Is there a config file somewhere that we could tuck the 240 into in case it needs adjusting on the fly?